EFCoreExercise/Exercises/Exercise14.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace EFCoreExercise.Exercises
+{
+    public class Exercise14
+    {
+        public List<string> OrderStudentsByName(string className)
+        {
+            using var dbContext = new DataContextFactory().CreateDbContext(new string[0]);
+
+            var innerJoin = from student in dbContext.Student
+                            join @class in dbContext.Class
+                            on student.Class.Id equals @class.Id
+                            select new { student, @class };
```

Comment:
Không cần thiết lắm

Chỉ cần
```suggestion
            var students = dbContext.Students.Where(x=>x.Class.Name == className).ToList()
```

Chú ý khai báo DbSet thì phải là số nhiều Students, Classes

Reply:
![image](https://user-images.githubusercontent.com/55595475/124406194-d2552780-dd6a-11eb-8aef-128d68998efe.png)
Em sửa như thế thì nó báo lỗi này anh ợ